apps/labs/posts/multidimensional-integration-in-scipy.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: 'Multdimensional integration in SciPy'
+description: Extending SciPy's integration facilities for multidimensional and array-valued integrands.
+authors: [olly-britton]
+published: September 12, 2024
+category: [PyData ecosystem, Internship, Array API]
+featuredImage:
+  src: /posts/multidimensional-integration-in-scipy/trapezoid-subdivisions-featured.png
+  alt: >
+    Applying the trapezoid rule to a curve with three trapeziums. The two trapeziums
+    on the right of the curve are darker and narrower, and their area more closely matches
+    the area of the function.
+hero:
+  imageSrc: /posts/multidimensional-integration-in-scipy/subdivisions-in-higher-dimensions.png
+  imageAlt: >
+    A plot of an undulating function in 3D. On top of the surface of the function,
+    there is a white mesh. The top right corner of the mesh is being filled in by a finer
+    mesh.
+---
+
+Hi! I'm Olly Britton, an undergraduate studying mathematics and computer science in the UK. This blog post talks about the work I did extending SciPy's integration facilities as part of a three-month internship with Quansight Labs.
+
+## The problem
+
+In one sentence, my work consisted of adding support for multidimensional numerical integration of array-valued functions into SciPy.
+
+Numerical integration, or "quadrature", is about _approximating_ the value of a definite integral of a function over some region – in other words, estimating the area or volume under a curve. This is in contrast to symbolic integration, which is interested in finding a closed-form for the _exact_ value.
+
+Before this internship, I wondered why you might ever want just an approximation rather than the exact value. But there are many good reasons to prefer numerical integration over symbolic integration: for one, the problem of finding out whether the integral of an arbitrary function has a "closed-form" is actually impossible in general! And even when it is, the results can be hopelessly complicated even for simple functions.
+
+{/*
+$$
+\begin{aligned}
+&\int^1_0 \frac{1}{1 + x + x^3} \, \text dx \approx 0.6303 \\\\
+&\int^1_0 \frac{1}{1 + x + x^3} \, \text dx = \frac{2 ^ \frac{4}{3}\cdot3 ^ \frac{5}{2} \left(\frac{\sqrt{31}}{2\cdot 3 ^ \frac{3}{2}} - \frac 1 2\right)\ln\Big(\Big| 2- \cdots}{\sqrt[3]{\sqrt{31} - 3 ^ \frac{3}{2}}\Big(\Big(-\frac{3 ^ \frac{3}{2} \sqrt[3]{29 \cdot 3 ^ \frac{3}{2}\cdot}}{\sqrt[3]{2}} \,\,\,\cdots}
+\end{aligned}
+$$
+*/}
```

Comment:
Based on an example from code search on the repository, 
https://github.com/search?q=repo%3AQuansight%2FQuansight-website%20%5Cfrac&type=code

it looks like TeX math needs a `math::` admonition, not sure. @melissawm would you be able to give us a hand please? Olly's post definitely needs TeX math but we're not sure how to make it render.

Reply:
I think this is actually TeX inside a docstring included in a code block.

Reply:
Indeed! So mu suggestion is useless, sorry for the noise! The question of how to use TeX math is still relevant though.

Reply:
Yeah, I think this is a blog setting, I'm really not sure 😦 Maybe someone from the website team can help - @pavithraes ?